### DISCUSSION TITLE
2D array in Postgres

### DISCUSSION THREAD

**Discussion by randomer** [prisma/prisma#6836] [Q&A] [answered]
Postgres can store 2D arrays but it looks like they can’t be represented with Prisma schema.
What is the best way to represent a 2D array with Prisma schema?

**Comment by austincrim** [comment, marked as answer]
Hey @randomer 👋 ,

I think the simplest way would be to use a JSON field type.

Would you mind opening a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for 2d array support?

Thanks!

**Reply by wxh06** [reply]
FYI, now there's a feature request for multi-dimensional array: #15676

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
